docs/application-development/testing.md

```diff
@@ -583,7 +679,18 @@ Content is currently unavailable.
 </TabItem>
 <TabItem value="python">
 
-Content is currently unavailable.
+To implement manual time skipping, use the [`start_time_skipping()`](https://python.temporal.io/temporalio.testing.WorkflowEnvironment.html#start_time_skipping) static method.
+
+```python
+from temporalio.testing import WorkflowEnvironment
+
+async def test_manual_time_skipping():
+    async with await WorkflowEnvironment.start_time_skipping() as env:
```

Comment:
It is important to note is that time auto-skips workflow tie and then stops auto-skipping during activity execution.